pkg/server/plugin/upstreamauthority/awssecret/awssecret_test.go

```diff
@@ -202,8 +218,11 @@ func (as *Suite) newAWSUpstreamAuthority(deprecatedTTL string) upstreamauthority
 }
 
 func (as *Suite) TestPublishJWTKey() {
-	resp, err := as.plugin.PublishJWTKey(ctx, &upstreamauthority.PublishJWTKeyRequest{})
-	as.Require().Nil(resp, "no response expected")
+	stream, err := as.plugin.PublishJWTKey(ctx, &upstreamauthority.PublishJWTKeyRequest{})
+	as.Require().Nil(err)
+	as.Require().NotNil(stream)
 
+	resp, err := stream.Recv()
```

Comment:
We should add an additional call to Recv() somewhere after here that we assert returns io.EOF.